csharp/src/Ice/Ice1Definitions.cs

```diff
@@ -134,7 +134,7 @@ internal static RetryPolicy GetRetryPolicy(IncomingResponseFrame response, Refer
                 {
                     istr = new InputStream(response.Payload.Slice(1),
                                            Ice2Definitions.Encoding,
-                                           reference.Communicator,
+                                           reference: reference,
```

Comment:
Why not just "reference"?

Reply:
See above. The InputStream ctor in question is:
```
 internal InputStream(
            ReadOnlyMemory<byte> buffer,
            Encoding encoding,
            Communicator? communicator = null,
            Connection? connection = null,
            Reference? reference = null,
            bool startEncapsulation = false)
```